.travis.yml

```diff
@@ -16,6 +16,12 @@ script:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
-
+  - "[[ $TRAVIS_BRANCH == 'master' ]] && [[ ${TRAVIS_PULL_REQUEST} == 'false' ]] &&
+    [[ ${TRAVIS_TAG} == '' ]] && mvn clean deploy --settings .travis.settings.xml"
 after_failure:
-  - if [ -f dubbo.log ]; then echo "------TAIL of dubbo.log------"; tail -n 1000 dubbo.log; echo "------END of dubbo.log------"; fi
+  - if [ -f dubbo.log ]; then echo "------TAIL of dubbo.log------"; tail -n 1000 dubbo.log;
+    echo "------END of dubbo.log------"; fi
```

Comment:
what is `dubbo.log`? maven logs?

Reply:
I don't know the purpose of `dubbo.log` for sure.

I think maybe all maven logs are stored in this file.